difftest/dpi_t1rocketemu/src/drive.rs

```diff
@@ -37,6 +41,218 @@ pub struct OnlineArgs {
 
   /// ISA config
   pub spike_isa: String,
+
+  /// DRAMsim3 configuartion (if any)
+  pub dramsim3_cfg: Option<PathBuf>,
```

Comment:
We need to force any downstream user to use it.
```suggestion
  pub dramsim3_cfg: PathBuf,
```

Reply:
Actually we don't. No dramsim config means that using the trivial latency model. This is a supported settings, but currently not easily settable. Switching the configuration to using plusargs should solve this.